rescheduler/rescheduler.go

```diff
@@ -24,15 +24,21 @@ import (
 	"time"
 
 	ca_simulator "k8s.io/contrib/cluster-autoscaler/simulator"
+	ca_drain "k8s.io/contrib/cluster-autoscaler/utils/drain"
```

Comment:
This was by design. Rescheduler shares logic with CA pretty much, because it performs similar simulations. The new dependency is just a result of refactoring in CA: some functions were move out of the simulator package to drain.